src/resmom/linux/mom_mach.h

```diff
@@ -110,7 +88,7 @@ typedef struct	pbs_plinks {		/* struct to link processes */
 	int	 pl_done;		/* kill has been done */
 } pbs_plinks;
 
-
+extern ulong totalmem;extern ulong totalmem;extern ulong totalmem;
```

Comment:
This looks incorrect.

Reply:
Whoops, I don't know why it repeats.  Sorry about that.  Fixed now.